docs/source/connectors.rst

```diff
@@ -0,0 +1,113 @@
+Connectors
+==========
+Connect your preprocessing pipeline with your favorite data storage platforms, and batch process all your documents using the provided CLI to store structured outputs locally on your filesystem. 
+
+To use any connector you'll need to first install ``local-inference``. You can find more information about this in the `installation guide <https://unstructured-io.github.io/unstructured/installing.html>`_.
+
+You can then use any connector with the ``unstructured-ingest`` command in the terminal. For example, the following command processes all the documents in S3 in the utic-dev-tech-fixtures bucket with a prefix of small-pdf-set/
+
+.. code:: shell
+
+  unstructured-ingest \
+    --remote-url s3://utic-dev-tech-fixtures/small-pdf-set/ \
+    --s3-anonymous \
+    --structured-output-dir s3-small-batch-output \
+    --num-processes 2
+
+To run this example, you'll first need to install the S3 dependencies as shown `here <https://unstructured-io.github.io/unstructured/connectors.html#s3-connector>`_.
```

Comment:
double check that link. it 404'd for me

Reply:
I think you're good to leave that in here

Reply:
ahh right 🤦 was looking at this too late at night. yes, totally